packages/generator/test/service-mapping.spec.ts

```diff
@@ -14,7 +14,7 @@ describe('service-mapping', () => {
         enumTypes: [],
         entities: [],
         functionImports: [],
-        namespace: 'fghjkl',
+        namespaces: ['fghjkl'],
```

Comment:
I know we had this here before, by why this name. If it's easy to fix consider replacing it with a better name ;)